Tests/UITests/TestApp/HealthKitTestsView.swift

```diff
@@ -13,8 +13,9 @@ import XCTSpezi
 
 
 struct HealthKitTestsView: View {
-    @EnvironmentObject var healthKitComponent: HealthKit<TestAppStandard>
-    @EnvironmentObject var standard: TestAppStandard
+    @EnvironmentObject var healthKitComponent: HealthKit
+    @EnvironmentObject var standard: ExampleStandard
+    
     @State var dataChanges: [String] = []
```

Comment:
Referencing my comment on `Tests/UITests/TestApp/TestApp.swift`, I would suggest that you extract the sample type (https://developer.apple.com/documentation/healthkit/hksample/1615744-sampletype) here and display that in the UI (e.g. using the `identifier` property: https://developer.apple.com/documentation/healthkit/hkobjecttype/1615294-identifier). 

This should show the element type in the UI and allows the UI tests to pick this up 👍